server/src/controllers/campaign.js

```diff
@@ -0,0 +1,97 @@
+const db = require("../../database/models/index");
+const BadRequestError = require("../errors/bad-request-error");
+const Op = db.Sequelize.Op;
+
+class CampaignController {
+  static getCampaigns = async (req, res) => {
+    const user = await db.User.findOne({
+      where: {
+        id: req.params.userId
+      }
+    });
```

Comment:
If we are using the requireAuth middleware, it already fetches the user... instead you should just be checking the req.user's id with the req.params.userId

Reply:
updated to use req.currentUser instead of req.params.userId